src/protobuf.zig

```diff
@@ -62,6 +87,7 @@ pub const ManagedString = union(ManagedStringTag) {
         }
     }
 
+    /// Creates a deep copy of the managed string using the provided allocator
```

Comment:
```suggestion
    /// Creates a copy of the managed string using the provided allocator
```
There are not shallow or deep copies in []u8